Library/Homebrew/os/mac/mach.rb

```diff
@@ -164,11 +164,10 @@ def initialize(path)
     end
 
     def parse_otool_L_output
-      ENV["HOMEBREW_MACH_O_FILE"] = path.expand_path.to_s
```

Comment:
I'm hesitant to remove this environment variable unless you're 100% sure there's no reason it was being used before.